relational_operators/TextScanOperator.cpp

```diff
@@ -18,6 +18,11 @@
  **/
 
 #include "relational_operators/TextScanOperator.hpp"
+#include "relational_operators/RelationalConfig.h"
```

Comment:
@hbdeshmukh Alternatively, I suggest to add an empty line between Line 20 and 21.